### PR TITLE
Just a few comments for new readers of the source

### PR DESCRIPTION
**Pull request by ekipan**
Take 2, as per https://github.com/jkotlinski/durexforth/pull/514#issuecomment-1416812515.

### CODE REVIEW

File: asm/durexforth.asm [hunk 1]
```diff
@@ -26,38 +40,38 @@ TIB = $200
...
+!set __LATEST = WORDLIST_BASE
+!macro BACKLINK .name , .namesize {
+    !set .xt = *
+    * = __LATEST - len(.name) - 3
+    !set __LATEST = *
+    !byte .namesize
+    !text .name
+    !word .xt
+    * = .xt
+}
```
Comment (ekipan) [inline asm/durexforth.asm:73]: While moving this macro I also changed the definition. I hope it's clearer. Not that it wasn't really before.

**Comment by jkotlinski** [comment]
Thank you!

**Comment by jkotlinski** [comment]
By the way. What did you think of the idea to move durexforth.asm to root folder, to make it easier for new users to find the entry point?

**Comment by ekipan** [comment]
I'm torn. On the one hand, yeah, it makes it more visible but on the other it's nice to have the sources together. Don't know if I would.

**Pull request merged**